public/js/embedListeners.js

```diff
@@ -175,21 +175,21 @@ function startListeners() {
 
         event.preventDefault()
         var deltaY = event.wheelDelta ? event.wheelDelta : -event.detail;
-        var scrolledUp = deltaY < 0;
-        var scrolledDown = deltaY > 0;
+        var scrolledDown = deltaY < 0;
+        var scrolledUp = deltaY > 0;
 
         if (event.ctrlKey) {
-            if (scrolledUp && globalScope.scale > 0.5 * DPR) {
+            if (scrolledDown && globalScope.scale > 0.5 * DPR) {
                 changeScale(-.1 * DPR);
             }
-            if (scrolledDown && globalScope.scale < 4 * DPR) {
+            if (scrolledUp && globalScope.scale < 4 * DPR) {
```

Comment:
You shouldn't have changes this as you have already inverted the values above.
Please revert back to `scrolledDown` only.